### PR TITLE
Fixing privacy_violation for HUB-1397

### PR DESCRIPTION
**Pull request by Haripriya99j**
Jira:
https://cirium.atlassian.net/browse/HUB-1397

Build link:
https://ddt-jenkins.prod.flightstats.io/job/run-hub-development-integration/161/

tests summary:
Integration tests:
https://ddt-jenkins.prod.flightstats.io/job/run-hub-integration-tests-against-cluster/102/console

System tests:
https://ddt-jenkins.prod.flightstats.io/job/run-hub-system-tests/25/

### CODE REVIEW

File: src/main/java/com/flightstats/hub/dao/aws/AwsConnectorFactory.java [hunk 1]
```diff
@@ -93,15 +93,16 @@ public AmazonDynamoDB getDynamoClient() {
     private AWSCredentialsProviderChain getAwsCredentials() {
         return new AWSCredentialsProviderChain(
                 new DefaultAWSCredentialsProviderChain(),
-                new AWSStaticCredentialsProvider(loadTestCredentials(awsProperties.getCredentialsFile())));
+                new AWSStaticCredentialsProvider(loadTestCredentials(awsProperties.getCredentialsFile()))
+        );
```
Comment (harinadh-dasari) [inline src/main/java/com/flightstats/hub/dao/aws/AwsConnectorFactory.java:97]: what is the difference here

Comment (Haripriya99j) [reply src/main/java/com/flightstats/hub/dao/aws/AwsConnectorFactory.java:97]: I just re-aligned with the brackets. You can see I just moved 1 closing bracket down and kept that colon beside it.

Original:

new AWSStaticCredentialsProvider(loadTestCredentials(awsProperties.getCredentialsFile())));

Modified:

new AWSStaticCredentialsProvider(loadTestCredentials(awsProperties.getCredentialsFile()))
        );
        
However it'not impacting anything, just aligned.

**Review by allustag — DISMISSED** [review]

**Comment by harinadh-dasari** [comment]
LGTM

**Review by harinadh-dasari — APPROVED** [review]
LGTM

**Review by allustag — APPROVED** [review]

**Pull request merged**